pom.xml

```diff
@@ -109,6 +108,22 @@
             </resource>
         </resources>
         <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Import-Package>
```

Comment:
Rather use `import-package` `export-package` maven properties than overriding the plugin configuration